application-load-balancer/main.tf

```
@@ -45,11 +52,12 @@ resource "aws_lb_listener" "alb-listener" {
 }
 
 resource "aws_security_group" "alb-security-group" {
-  # checkov:skip=CKV2_AWS_5:Security group is used by VPC. Ticket to investigate: https://uktrade.atlassian.net/browse/DBTP-1039
-  for_each = local.protocols
-  name     = "${var.application}-${var.environment}-alb-${each.key}"
-  vpc_id   = data.aws_vpc.vpc.id
-  tags     = local.tags
+  # checkov:skip=CKV2_AWS_5: False Positive in Checkov - https://github.com/bridgecrewio/checkov/issues/3010
+  for_each    = local.protocols
+  name        = "${var.application}-${var.environment}-alb-${each.key}"
+  description = "Managed by Terraform"
```

Don't we already have a tag that tells us that this resource is managed by terraform?

```suggestion
  description = "Security group for application load balancer"
```

---

I'm going to revert this one as changing the description makes it want to recreate the security groups.